docs/docs/config/overview.md

```diff
@@ -25,14 +25,6 @@ sudo ./dist/tracee --config ./examples/config/global_config.yaml --log info
 
 The configuration file can be in any format supported by the [viper](https://github.com/spf13/viper) library, which includes YAML, JSON, TOML, INI, HCL and Java properties. The configuration file should contain a mapping of flag names to their values. For example, to output aggregated debug level logs every default seconds `--log debug --log aggregate`, you would add the following to your configuration file:
 
-```yaml
```

Comment:
we mentioned in the call yesterday that we only want to keep the structured output -- also, otherwise, configuring output and other options to pass into the CLI seems a lot more complex